templates/repo/branch/list.tmpl

```diff
@@ -4,7 +4,7 @@
 	<div class="ui container">
 		{{template "base/alert" .}}
 		{{template "repo/sub_menu" .}}
-		{{if .DefaultBranchBranch}}
+		{{if .DefaultBranch}}
```

Comment:
I think `DefaultBranchBranch` is still a good name, to show that it is a "Branch" structure.

Otherwise the naming is messy, `DefaultBranch` is a string in most cases.